subsys/disk/disk_access_spi_sdhc.c

```diff
@@ -795,8 +798,12 @@ static int sdhc_spi_init(struct device *dev)
 		.slave = DT_REG_ADDR(DT_INST(0, zephyr_mmc_spi_slot)),
 	};
 	/* SPI config for default speed */
+	if (spi_max_frequency > SDHC_SPI_DEFAULT_SPEED) {
```

Comment:
Why is this a MAX?  Shouldn't the driver always take the speed from DeviceTree if it is supplied?

Reply:
the commiter took the name out of the dts property (see line 789 above), but it is definitely coming from dts here.